source/mir/random/algorithm.d

```diff
@@ -34,868 +23,224 @@ Params:
     var = random variable (optional)
     lengths = one or more lengths
 +/
-auto randomSlice(T, G, D, size_t N)(G gen, D var, size_t[N] lengths...)
-    if (isSaturatedRandomEngine!G && isRandomVariable!D &&
+pragma(inline, false)
```

Comment:
Why `pragma(inline, false)`?

Reply:
It will generate quite a large chunk of code that may be frequently used in different places. The pragma helps to reduce code size. Mir based programs should be huge as vibe.d